firestore.rules

```diff
@@ -17,27 +17,40 @@ service cloud.firestore {
       allow read: if request.auth.token.user_type == 'teacher' && request.auth.token.class_hash == resource.data.context_id;
       allow read: if request.auth.token.user_type == 'learner' && string(request.auth.token.platform_user_id) == string(resource.data.platform_user_id);
       allow update: if request.auth.token.user_type == 'learner'
-              && string(request.auth.token.platform_user_id) == string(resource.data.platform_user_id)
+      				&& string(request.auth.token.platform_user_id) == string(resource.data.platform_user_id)
               && request.auth.token.class_hash == resource.data.context_id;
       allow create: if request.auth.token.user_type == 'learner'
-              && string(request.auth.token.platform_user_id) == string(request.resource.data.platform_user_id)
+      				&& string(request.auth.token.platform_user_id) == string(request.resource.data.platform_user_id)
               && request.auth.token.class_hash == request.resource.data.context_id;
       // anonymous requests based on run_key, which must be larger than 10 chars
-      allow read, update: if resource.data.run_key.size() > 10 && resource.data.run_key == request.resource.data.run_key;
-      allow create: if request.resource.data.run_key.size() > 10;
-    }  
+      allow read, update: if resource.data.run_key.size() > 10
```

Comment:
I think we disabled this check for testing while developing the anonymous report, and then we forgot to find a better way.  I added a note to this story about fixing this: https://www.pivotaltracker.com/story/show/174883954